sdk/core/Azure.Core/samples/Configuration.md

```diff
@@ -103,6 +103,55 @@ SecretClientOptions options = new SecretClientOptions()
 };
 ```
 
+Another customization that is possible is to easily configure what exceptions or status codes 
```

Comment:
Possibly silly question, but if the primary motivating scenario for this is extending/overriding the set of retriable status codes, would a simpler API be to provide a list of those status codes?

Reply:
I see, so you were interested in not just status codes, but also exceptions that indicated the need to retry?  Or were you interested in overriding IsErrorResponse here as well?  If the latter is true, I'm not sure the current approach will allow that because it's not used elsewhere in the pipeline.

Actually, thinking about this more, @jsquire, is there a reason you need the classifier to be localized to the retry policy? Or could you set one another way, like through the pipeline or on the message or request context?

Is this the first time we're making it possible for an end-user customer and not a library author to set response classifier?  Or, the first time outside of request context?  If so, we may want to consider those implications as well.

Reply:
Also, I'm wondering - given the complexity we're discovering, if this is needed internally for testing and not by customers, would it make sense to address it differently for this case?

Reply:
I think this is a clear gap in the functionality of RetryPolicy. A very common scenario is that users want to configure what will be retried. ResponseClassifier provides a super easy interface to accomplish this, and it is a building block of RetryPolicy.